airgun/entities/contenthost.py

```diff
@@ -195,3 +195,10 @@ def step(self, *args, **kwargs):
         entity_name = kwargs.get('entity_name')
         self.parent.search(entity_name)
         self.parent.table.row(name=entity_name)['Name'].widget.click()
+
+@navigator.register(ContentHostEntity, 'Read')
+class ReadContentHost(EditContentHost):
+    def post_navigate(self, _tries, *args, **kwargs):
```

Comment:
Just to let you know: this `post_navigate` should probably be in `EditContentHost` class.

Here, you register new navigation endpoint for ContentHostEntity called `Read`. That `post_navigate` would be run only if you do this anywhere in your entity:
```python
self.navigate_to(self, 'Read')
```

But since you never try to navigate to this endpoint, this method is never executed.

(This whole approach turned out to be incorrect, as you would check checkbox and then next `read()` call would switch tab and reset your checkbox state. But I wanted to point out this detail about navigation and relationship between entity and other classes in file.)

Reply:
Thanks for the clarification!